tests/acceptance/features/webUISharingInternalUsersToRootBlacklisted/shareWithUsers.feature

```diff
@@ -10,17 +10,6 @@ Feature: Sharing files and folders with internal users
       | Alice    |
       | Brian    |
 
-  @skip @yetToImplement
```

Comment:
why deleting this test?

Reply:
should cover by
https://github.com/owncloud/web/blob/e1b958326be41975cd53445a6d88bcf92f3be1e2/tests/acceptance/features/webUISharingInternalUsersToRootBlacklisted/shareWithUsers.feature#L58-L67